api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -483,4 +494,32 @@ public void validate_shouldFailIfCodedConceptContainsItselfAsAnAnswer() {
 		new ConceptValidator().validate(concept, errors);
 		Assert.assertEquals(true, errors.hasErrors());
 	}
+
+	/**
+	 * @see ConceptValidator#validate(Object,Errors)
+	 **/
+	@Test
+	@Verifies(value = "fail if any description is a not entered while creating a new concept ", method = "validate(Object,Errors)")
+	public void validate_shouldFailIfAnyDescriptionIsNotEnteredWhileCreatingANewConcept() throws Exception {
+		Concept concept = new Concept();
+		concept.addName(new ConceptName("some name", Context.getLocale()));
+		Errors errors = new BindException(concept, "concept");
+		new ConceptValidator().validate(concept, errors);
+		Assert.assertTrue(errors.hasFieldErrors("descriptions"));
+	}
+
+	/**
+	 * @see ConceptValidator#validate(Object,Errors)
+	 **/
+	@Test
+	@Verifies(value = "should pass if  none of the concept descriptions is null", method = "validate(Object,Errors)")
+	public void validate_shouldPassIfNoneofTheConceptDescriptionsIsNull() throws Exception {
+		Concept concept = new Concept();
+		concept.addName(new ConceptName("some name",Context.getLocale()));
+		concept.addDescription(new ConceptDescription("some description",null));
+		Errors errors = new BindException(concept, "concept");
+		new ConceptValidator().validate(concept, errors);
+		Assert.assertFalse(errors.hasErrors());
+	}
+
```

Comment:
I also expected to see a test that shows that validation fails when the only description added is an empty string